packages/static-renderer/src/pm/markdown/markdown.ts

```diff
@@ -134,9 +135,9 @@ export function renderToMarkdown({
         highlight({ children }) {
           return `==${serializeChildrenToHTMLString(children)}==`
         },
-        ...options?.markMapping,
+        ...markMapping,
       },
-      ...options,
```

Comment:
This overrides `nodeMapping` and `markMapping` entirely as opposed to allow extending each